shared/js/ui/models/site.es6.js

```diff
@@ -248,7 +248,20 @@ Site.prototype = window.$.extend({},
                 this.isWhitelisted = !this.isWhitelisted
                 this.set('whitelisted', this.isWhitelisted)
                 const whitelistOnOrOff = this.isWhitelisted ? 'off' : 'on'
-                this.fetch({ firePixel: ['ept', whitelistOnOrOff] })
+
+                // fire ept.on pixel if just turned privacy protection on,
+                // fire ept.off pixel if just turned privacy protection off.
+                if (whitelistOnOrOff === 'on') {
+                    // If user reported broken site and opted to share data on site,
+                    // attach domain to ept.on pixel if they turn privacy protection back on.
+                    if (this.whitelistOptIn) {
+                        this.fetch({ firePixel: ['ept', 'on', this.tab.site.domain] })
+                    } else {
+                        this.fetch({ firePixel: ['ept', 'on'] })
+                    }
+                } else {
+                    this.fetch({ firePixel: ['ept', 'off'] })
```

Comment:
super minor nit, but I wonder if it's worth having 'ept' in a variable, since it's repeated multiple times, so if the name changes you'd only have to replace it on one line :)